packages/css-framework/src/components/_container.scss

```diff
@@ -2,16 +2,16 @@
   max-width: $navy-content-width;
   margin-left: auto;
   margin-right: auto;
-  padding-left: spacing(3);
-  padding-right: spacing(3);
```

Comment:
Wow thats a big change

Reply:
`spacing(3)` is `0.75rem`. That's nowhere near enough padding.

Reply:
If we think we need it that small for apps, then I can put it back. _However_, we categorically need more padding on the docs site, so I'd suggest we create a completely separate container for the docs.